static/templates/change_stream_info_modal.hbs

```diff
@@ -0,0 +1,34 @@
+<div id="change_stream_info_modal" class="modal modal-bg hide fade new-style" tabindex="-1"
+  role="dialog" aria-labelledby="change_stream_info_label" aria-hidden="true">
+    <div class="modal-header">
+        <button type="button" class="close close-change-stream-info-modal" data-dismiss="modal" aria-label="{{t 'Close' }}">
+            <span aria-hidden="true">&times;</span>
+        </button>
+        <h3 id="change_stream_info_label">
+            {{t "Change stream info for #" }}{{ stream_name }}
```

Comment:
Same comment on translation tags and word order here.  See https://zulip.readthedocs.io/en/latest/translating/internationalization.html for background.

Reply:
Made this change. Also added an extra commit for fixing similar issue in `subscription_stream_privacy_modal.hbs`.